Makefile

```diff
@@ -116,9 +116,12 @@ cover:
 cover-viz: cover
 	$(GO) tool cover -html=build/coverage.out
 
-# Compile Pilosa
+# Build featurebase
 build:
 	$(GO) build -tags='$(BUILD_TAGS)' -ldflags $(LDFLAGS) $(FLAGS) ./cmd/featurebase
+
```

Comment:
If separating to a separate build target please ensure the package target builds fbsql so it gets included in cloud environments